utils/strings/validations.ts

```diff
@@ -0,0 +1,3 @@
+export function isEmpty(value: string) {
+  return !value || !value.trim();
```

Comment:
```suggestion
  return !value.trim();
```
Minor nit since this covers both cases